js/RateCurve.js

```diff
@@ -75,8 +79,7 @@ RateCurve.prototype.rcCommandRawToDegreesPerSecond = function (rcData, rate, rcR
             angleRate = (rate + 27) * inputValue / 16;
```

Comment:
The rate  calculation was never updated to reflect changes since the first implementation, so it did not get any of the changes that were made to the 'Betaflight' model only, and it still uses the same model for both 'Betaflight' and 'Legacy'.

If we need more accurate modelling we will need to change this, but since it's only really used for the rates curve and the 3d model, I don't know if absolute accuracy is needed for this, or if something simpler is good enough for the purpose?